api/v1alpha1/virtualmachineclassbinding_types.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClassReference contains info to locate a Kind VirtualMachineClass object
+type ClassReference struct {
+	// Name is the name of resource being referenced.
```

Comment:
I think we should add the `ApiVersion` or `ApiGroup` and `Kind` since we want to make the association to a particular VirtualMachineClass explicit. `Kind` is sort of redundant, but I would rather have the association explicit than have a webhook default to an older version once we upgrade to a newer version.

Reply:
I've made them available.